client/landing/stepper/declarative-flow/internals/steps-repository/playground/lib/initialize-playground.ts

```diff
@@ -0,0 +1,195 @@
+import config from '@automattic/calypso-config';
+import { Blueprint } from '@wp-playground/blueprints';
+import { MountDescriptor, PlaygroundClient, startPlaygroundWeb } from '@wp-playground/client';
+import { logToLogstash } from 'calypso/lib/logstash';
+import { resolveBlueprintFromURL } from './resolve-blueprint-from-url';
+
+const OPFS_PATH_PREFIX = '/wpcom-onboarding';
+
+// Blueprint management is encapsulated in this closure
+const blueprintManager = ( () => {
+	const DEFAULT_BLUEPRINT: Blueprint = {
+		preferredVersions: {
+			php: '8.3',
```

Comment:
Should we use `usePhpVersions().recommendedValue` here instead of hardcoding 8.3?

Reply:
It's always overwritten with the specified recommended php version value. Had a comment on that line which got removed. Reason why I wrapped both constants in its closure, was to prevent the direct usage. I guess we can remove it altogether or make it act as a fallback when an invalid value for recommendedPhpVersion is supplied.